scripts/setup-node.sh

```diff
@@ -108,6 +108,7 @@ parse_params() {
     bin_directory="$root_directory/bin"
     logs_directory="$root_directory/logs"
     config_directory="$root_directory/config"
+    share_lib_directory="$root_directory/lib"
```

Comment:
did you test resuming a syncing node?

Reply:
stagenet hasn't been upgraded yet. so you'll need to try syncing once it's updated to ensure the local node can actually sync wasm blocks